Tests/SparkConnectTests/DataFrameTests.swift

```diff
@@ -936,8 +938,9 @@ struct DataFrameTests {
     @Test
     func timestamp() async throws {
       let spark = try await SparkSession.builder.getOrCreate()
+      // TODO(SPARK-52747)
```

Comment:
Timestamp support is not released yet. I created SPARK-52747 to investigate timezone issue completely.

Reply:
Okay. Thanks @dongjoon-hyun